updating/understanding-upgrade-channels-release.adoc

```diff
@@ -19,6 +19,11 @@ ifndef::openshift-origin[]
 
 If you do not want the Cluster Version Operator to fetch available updates from the upgrade recommendation service, you can use the `oc adm upgrade channel` command in the OpenShift CLI to configure an empty channel. This configuration can be helpful if, for example, a cluster has restricted network access and there is no local, reachable upgrade recommendation service.
 
+[WARNING]
+====
+Red Hat reccomends contiguous versions. Upgrading to non-contiguous version will not guarantee the backward compatibility as RH does not test it.
```

Comment:
Most of this is just style guide stuff. Also suggested adding "only" to make the recommendation a bit more clear, but obviously I defer to the SMEs here on this :slightly_smiling_face: 
```suggestion
Red Hat recommends upgrading to contiguous versions only. Red Hat does not test upgrades to noncontiguous versions and cannot guarantee compatibility with earlier versions.
```